examples/react-widget-app/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@buildwithsygma/react-widget-app",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "yarn run lint:types && yarn run lint:code",
+    "lint:types": "tsc -p ./tsconfig.json --noEmit",
+    "lint:code": "eslint '{src,tests}/**/*.ts'",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@buildwithsygma/sygmaprotocol-react-widget": "workspace:^",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
+    "@chainsafe/eslint-config": "2.2.2",
+    "@types/react": "^18.2.43",
+    "@types/react-dom": "^18.2.17",
+    "@typescript-eslint/eslint-plugin": "^6.14.0",
+    "@typescript-eslint/parser": "^6.14.0",
```

Comment:
you can remove @typescript-eslint/* packages, they get installed via @chainsafe/eslint-config

Reply:
done